services/validation/duty_validation.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  CannotModifyAssignedDutyError,
+  DocumentNotFoundError,
+} from '../../error_handling/client_errors';
+import { Duty } from '../schemas/duty.zschema';
+
+const validateDutyIsModifiable = (id: string, duty: Duty | null) => {
+  if (duty && duty.soldiers?.length && duty.soldiers?.length > 0) {
```

Comment:
duty.soldiers.length will return 0, which is falsy.

Reply:
good point

Reply:
resolved